### DISCUSSION TITLE
How can I reload current screen only ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [orchidsoftware/platform#2468] [Q&A]
As orchid is SPA if there is a way with JS event (selection option selected ) to reload current screen, 
with different content depending on selection option selected (like saving key data in session and reading it on screen reload).
I mean not to reload all page with js command like :

`location.reload() 
`
but only screen part? Or maybe part of this  screen part ?

If yes, please examples or ref to docs.

```
        "laravel/framework": "^9.19",
        "orchid/crud": "^3.8",
        "orchid/platform": "^13.6",

```
Thanks!

**Comment by sergeynilov** [comment]
Which version of Stimulus is used in orchid ?
Are there some more tutorials/docs on using Stimulus  in orchid ?
I read this 
https://orchid.software/en/docs/javascript/
But it is too small info...

**Reply by tabuna** [reply]
We are using version 3. You can see the versions used in the package.json file

https://github.com/orchidsoftware/platform/blob/6db605be1f10d46a682b2f60e1f748acbc9f9d84/package.json#L17-L19

**Reply by sergeynilov** [reply]
No in my  package.json file I have : 

```
{
    "private": true,
    "scripts": {
        "dev": "vite",
        "build": "vite build"
    },
    "devDependencies": {
        "axios": "^0.27",
        "laravel-vite-plugin": "^0.6.0",
        "lodash": "^4.17.19",
        "postcss": "^8.1.14",
        "vite": "^3.0.0",
        "vite-plugin-static-copy": "^0.12.0"
    }
}
 
```
I did not remove any packages from this file...

Also, please, more tutorials/docs on using Stimulus in orchid ?
Where to start from ? Stimulus entry point?

**Reply by c-gross** [reply]
You checked the package.json in your root folder. To see the check files in orchid you have to check vendor/orchid/platform.
For the package.json it's vendor/orchid/platform/package.json
